### PR TITLE
Driver Availability: Added Navbar

### LINKED ISSUE #52: DriverAvailability: Driver Availability Navbar
## Acceptance Criteria: Driver Perspective

- [ ] Any driver (ROLE_DRIVER) should be able to see a menu option `Availability` under `Shifts` on the navbar.

### Acceptance Criteria: Admin Perspective

- [ ] Under the admin option of the navbar there is an option `Driver Availability` where they can see all drivers' availability.

### PR DESCRIPTION
**Pull request by Rytuose**
We added a navbar for driver availability so admins and drivers can access the index, edit, and create pages for driverAvailability. The current pages are placeholders.

## Acceptance Criteria: Driver Perspective

- [x] Any driver (ROLE_DRIVER) should be able to see a menu option `Availability` under `Shifts` on the navbar.

### Acceptance Criteria: Admin Perspective

- [x] Under the admin option of the navbar there is an option `Driver Availability` where they can see all drivers' availability.

DriverAvailabilityNavbarCreate
DriverAvailabilityNavbarEdit
DriverAvailabilityNavbarIndex

Closes #52

### CODE REVIEW

**Review by calee14 — APPROVED** [review]
LGTM

**Review by cydthe — APPROVED** [review]
LGTM

File: frontend/src/App.js [hunk 1]
```diff
@@ -108,6 +112,19 @@ function App() {
         {
           (hasRole(currentUser, "ROLE_MEMBER")) && <Route exact path="/apply/rider/edit/:id" element={<RiderApplicationEditPageMember />} />
         }
+        {
+          (hasRole(currentUser, "ROLE_ADMIN")) && <Route exact path="/admin/driverAvailability/" element={<DriverAvailabilityCreatePage />} />
+        }
+        {
+          (hasRole(currentUser, "ROLE_ADMIN") || hasRole(currentUser, "ROLE_DRIVER") || hasRole(currentUser, "ROLE_RIDER")) && <Route exact path="/driverAvailability/" element={<DriverAvailabilityIndexPage />} />
```
Comment (andrewpengucsb) [inline frontend/src/App.js:119]: Regular riders should not be able to see when drivers are available

File: frontend/src/main/components/Nav/AppNavbar.js [hunk 2]
```diff
@@ -88,10 +88,18 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
                 </NavDropdown>
                 )
               }
+              {
+                hasRole(currentUser, "ROLE_ADMIN") && (
+                  <NavDropdown title="Driver Availability" id="appnavbar-driverAvailability-dropdown" data-testid="appnavbar-driverAvailability-dropdown" >
+                  <NavDropdown.Item as={Link} to="/driverAvailability">Dashboard</NavDropdown.Item>
```
Comment (andrewpengucsb) [inline frontend/src/main/components/Nav/AppNavbar.js:94]: Dashboard is incorrect, probably forgot to change when copying?

**Review by andrewpengucsb — CHANGES_REQUESTED** [review]

**Comment by shamitagurusu** [comment]
LGTM 👍

**Review by andrewpengucsb — APPROVED** [review]
LGTM

**Comment by shamitagurusu** [comment]
LGTM 👍 👍

**Review by shamitagurusu — APPROVED** [review]

**Pull request merged**